.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     hooks:
       - id: pants-fmt-lint
         name: pants fmt lint
-        always_run: false
+        always_run: true
```

Comment:
I thought this might fix the precommit flakey linter but no dice, so I filed an issue https://github.com/mozilla-ai/lumigator/issues/213